### ISSUE TITLE
`GetFileFromApplicationUriAsync` not works as 5.4 and earlier at 5.5 and net9

### ISSUE THREAD

**Issue by SongOfYouth** [unoplatform/uno#18761]
It raised a "file can not be found" error when using **GetFileFromApplicationUriAsync** to load a content file,but it's ok on net8 and uno 5.5.32.

**Comment by SongOfYouth** [comment]
> Thanks for the report. Can you provide a repro?

[GetFileDemo.zip](https://github.com/user-attachments/files/17694295/GetFileDemo.zip)

**Comment by jeromelaban** [comment]
Thanks. This issue has been fixed in a later version of the bootstrapper. Could you try adding:

```xml
<PropertyGroup>
<UnoWasmBootstrapVersion>9.0.4</UnoWasmBootstrapVersion>
</PropertyGroup>
```

We'll be releasing soon a new version of the uno.sdk that uses this version.

**Comment by SongOfYouth** [comment]
> Thanks. This issue has been fixed in a later version of the bootstrapper. Could you try adding:
> 
> ```
> <PropertyGroup>
> <UnoWasmBootstrapVersion>9.0.4</UnoWasmBootstrapVersion>
> </PropertyGroup>
> ```
> 
> We'll be releasing soon a new version of the uno.sdk that uses this version.

I've tried it, not work. deleted obj and bin folder not work too. maybe i should wait for your new version of uno.sdk.

**Comment by jeromelaban** [comment]
Thanks for the update. We'll take a look when possible.

**Comment by jeromelaban** [comment]
This was fixed in 9.0.4, but it could be that the update you made did not work.

You can upgrade to the uno.sdk 5.5.43 to fix the issue.
